.github/workflows/ci.yml

```diff
@@ -20,6 +20,8 @@ jobs:
       run: pnpm install
     - name: Build
       run: pnpm build
+    - name: Format
```

Comment:
Should be a seperate workflow

Reply:
Can you please explain more? I've not written a github action before. Can it not be under Ubuntu, Windows, or Mac?

Reply:
Do you mean a separate yml file?

Reply:
> Can you please explain more? I've not written a github action before. Can it not be under Ubuntu, Windows, or Mac?

No it should be under a seperate one labelled, Format